client/src/components/job-applications-view-component/job-application-table-component/index.jsx

```diff
@@ -0,0 +1,163 @@
+//    match the correct URLs
+// 2. When s3 resume download feature is complete, add download link to corresponding column
+
+function JobApplicationsTable({
+  jobApplications,
+  totalJobApplicationsNumber,
+  page,
+  rowsPerPage,
+  handlePageRowChange,
+}) {
+  const formatPhoneNumber = (phoneNumber) =>
+    phoneNumber.replace(/(\d{3})(\d{3})(\d{4})/, "$1-$2-$3");
+
+  const formatPostalCode = (postalCode) =>
+    postalCode
+      .toUpperCase()
+      .replace(/^([A-Za-z]\d[A-Za-z])(\d[A-Za-z]\d)$/, "$1 $2");
+
+  return (
+    <TableContainer
+      sx={{
+        marginTop: "35px",
+        width: "100%",
+        padding: "10px 2%",
+        boxSizing: "border-box",
+      }}
+      component={Paper}
+    >
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell sx={{ fontWeight: 700, width: "8%" }} align="left">
+              Job ID #
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "15%" }} align="left">
+              Title
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "12.5%" }} align="left">
+              Applicant Name
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "12.5%" }} align="left">
+              Email
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "10%" }} align="left">
+              Phone
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "10%" }} align="left">
+              Postal Code
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "10%" }} align="left">
+              Date Applied
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "10%" }} align="left">
+              Status
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "8%" }} align="left">
+              Resume
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {jobApplications.map((row) => (
+            <TableRow key={row.id}>
+              <TableCell align="left">{row.id}</TableCell>
+              <TableCell align="left">
+                <Link href={`/job-posts/${row.job_posting_id}`}>
+                  {row.title}
+                </Link>
+              </TableCell>
+              <TableCell align="left">
+                <Link href={`/job-applications/${row.id}`}>{row.name}</Link>
+              </TableCell>
+              <TableCell align="left">{row.email}</TableCell>
+              <TableCell align="left">{formatPhoneNumber(row.phone)}</TableCell>
+              <TableCell align="left">
+                {formatPostalCode(row.postal_code)}
+              </TableCell>
+              <TableCell align="left">
+                {/* Expect createdAt to be formated as date for ease of formatting */}
+                {row.createdAt.toLocaleDateString("en-CA")}
```

Comment:
for the sake of consistency, please use the date formatter we have! client/src/utils/date.js